tests/helper/helper_kubectl.go

```diff
@@ -145,3 +145,14 @@ func (kubectl KubectlRunner) GetEnvsDevFileDeployment(componentName string, proj
 	}
 	return mapOutput
 }
+
+// DeleteLocalConfig helps user to delete local config files with flags
+func (kubectl KubectlRunner) DeleteLocalConfig(args ...string) {
+	CmdShouldFail("odo", args...)
+	output := CmdShouldPass("odo", append(args, "-af")...)
+	expectedOutput := []string{
+		"Successfully deleted env file",
+		"Successfully deleted devfile.yaml file",
+	}
+	MatchAllInOutput(output, expectedOutput)
+}
```

Comment:
This is a odo specific function and the oc runner version is exactly same as this one. This should be in the helper file.

Reply:
The best appropriate location I could look for is `utils` file. So moved it there.